src/parameters.ts

```diff
@@ -24,3 +24,11 @@ export interface GetChunkDetailsParams {
   block_id?: BlockTag | Promise<BlockTag>
   shard_id?: number
 }
+
+export interface GetStateParams {
+  request_type: 'view_state'
+  finality?: Finality
+  block_id?: BlockTag | Promise<BlockTag>
+  account_id: string | Promise<string>
```

Comment:
the params to be sent should never be of `Promise` type. Those promises should be resolved before being passed as params

Reply:
You're right, thanks!